internal/app/grpcserver/impl.go

```diff
@@ -62,11 +62,12 @@ func (c *conformanceServiceServer) Unary(
 		if err := grpc.SetTrailer(ctx, trailerMD); err != nil {
 			return nil, err
 		}
-		time.Sleep((time.Duration(req.ResponseDefinition.ResponseDelayMs) * time.Millisecond))
+		time.Sleep(time.Duration(req.ResponseDefinition.ResponseDelayMs) * time.Millisecond)
```

Comment:
This change ("remove redundant parentheses") was suggested by GoLand, so I went ahead and accepted here and throughout.